sim/src/main/cc/firesim/firesim_top.cc

```diff
@@ -389,56 +394,64 @@ firesim_top_t::firesim_top_t(int argc, char** argv)
     add_bridge_driver(new synthesized_assertions_t(this,
```

Comment:
As above, you can delete the enable asserts parameter once you move the plusarg handling into the bridge driver.